src/oaklib/interfaces/dumper_interface.py

```diff
@@ -27,7 +27,7 @@ class DumperInterface(BasicOntologyInterface, ABC):
     An OntologyInterface that is capable of exporting entire contents.
     """
 
-    def dump(self, path: str = None, syntax: str = None):
+    def dump(self, path: str = None, syntax: str = None, **kwargs):
```

Comment:
@joeflack4 - note how the *interface* accepts an open-ended list of params. Different syntaxes, and different implementations may have different such params